proposed/2021/DotNetAudit.md

```diff
@@ -0,0 +1,296 @@
+# dotnet audit & dotnet audit fix
+
+- [Jon Douglas](https://github.com/JonDouglas/)
+- Start Date (2021-04-01)
+- [#8087](https://github.com/NuGet/Home/issues/8087)
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+`dotnet audit` & `dotnet audit fix` helps you find, fix, and monitor known security vulnerabilities, deprecated packages, and outdated versions in your .NET projects & solutions.
+
+## Motivation
```

Comment:
In my time in the NuGet team, there have been a few times when we've been looking at some old feature and trying to understand WHY it was done a particular way. So, in the interest of helping people in the future understand, can you please add something (anywhere in the document, not necessarily here) about why we're not just adding a `--fix` argument to `dotnet list package --deprecated --include-transitive`?

Reply:
Sure thing.